sds/src/scanner/mod.rs

```diff
@@ -186,6 +186,7 @@ pub struct Scanner {
     cache_pool: CachePool,
     scanner_features: ScannerFeatures,
     metrics: ScannerMetrics,
+    multipass_v0_enabled: bool,
```

Comment:
I think we want to have this part of `ScannerFeatures`, so avoid having a giant list of attributes for configs

Reply:
Sure, I can change it. The `ScannerFeatures` were mostly useful before we had the `ScannerBuilder` since you had to pass in all options. Now the new feature is set by the builder, so you don't pass it in at all unless you want to change it from the default. (This change won't change anything use facing)

Reply:
https://github.com/DataDog/dd-sensitive-data-scanner/pull/93/commits/93c975d3298f26a6adc789dd8c0fbee102a450d0

Reply:
Agree, For bindings this should simplify a bit the code by forwarding a byte array of ScannerFeatures instead of having to forward each argument independently.